addons/web_calendar/static/src/js/web_calendar.js

```diff
@@ -892,7 +892,9 @@ var CalendarView = View.extend({
         var index = this.dataset.get_id_index(id);
         if (index !== null) {
             event_id = this.dataset.ids[index];
-            this.dataset.write(event_id, data, {}).always(function() {
+            var context = {};
+            context.from_ui = true;
```

Comment:
Why not just pass `{context: {from_ui: true}}` in directly?

Reply:
So you get time to react to my commit :dancer: